cfme/middleware/server.py

```diff
@@ -233,6 +233,11 @@ def server_in_rest(self):
         raise NotImplementedError('This feature not implemented yet')
 
     @variable(alias='ui')
+    def is_reload_required(self):
```

Comment:
@psav those methods for those detail checks seem to be rather uniform, i wonder if a helper might make sense